developer/testing/release-process.md

```diff
@@ -70,6 +70,13 @@ Notes:
 - Wait for all `master` tests to pass.
 - Follow the format of previous release, copy change log from release PR into the release notes.
 
-## Cleanup
+## Post-release checklist: All releases
 
-- Review issues that the release resolves, that they are closed and stale branches removed.
+- Core team: Review issues that the release resolves, that they are closed and stale branches removed.
+- Community team: Update Gitter with links to the latest release, changelog and update instructions.
+
+## Post-release checklist: Major releases
+
+- Community team: Tag a new Documentation release and update Editorial about any new documentation tutorial links, new FAQ questions and other relevant content for social media channels.
+- Community / Editorial team: If a release is noteworthy enough for a blog post, assign a blog post writer for the release post. The post should include instructions on how to upgrade.
+- Community team: Assign team members to review all [major plugin tutorial repositories](https://github.com/reactioncommerce/reaction-example-plugin/), [YouTube videos](https://www.youtube.com/playlist?list=PLJ1TVRVOrm2O5OsXqzDn5iZez4WEnKRZH), [`reaction-devtools`](https://github.com/reactioncommerce/reaction-devtools), [`reaction-swag-shop`](https://github.com/reactioncommerce/reaction-swag-shop) and documentation tutorials against the latest release. If plugins or tutorials fail, make issues appropriately.
```

Comment:
That link says "major plugin tutorial repositories" like it will be more than one but just links to the single repo. Can we just keep an explicit list (which I think you have most things there). Or maybe point to the "Supported Community plugins" page?